src/Native/Unix/System.Security.Cryptography.Native.Apple/pal_rsa.cpp

```diff
@@ -7,8 +7,8 @@
 static int32_t ExecuteCFDataTransform(
     SecTransformRef xform, uint8_t* pbData, int32_t cbData, CFDataRef* pDataOut, CFErrorRef* pErrorOut);
 
-extern "C" int
-AppleCryptoNative_RsaGenerateKey(int32_t keySizeBits, SecKeyRef* pPublicKey, SecKeyRef* pPrivateKey, int32_t* pOSStatus)
+extern "C" int AppleCryptoNative_RsaGenerateKey(
```

Comment:
`int` => `int32_t`... maybe search and replace in these files in case there are more of 'em